scd/usbtoken.c

```diff
@@ -353,31 +381,29 @@ usbtoken_new(const char *serial)
 
 	int rc;
 	usbtoken_t *token = NULL;
+	unsigned char *brsp = NULL;
+
+	brsp = mem_new0(unsigned char, ICC_ATR_BUF_LEN);
+	IF_NULL_RETVAL_ERROR(brsp, NULL);
+	size_t brsp_len = ICC_ATR_BUF_LEN;
 
 	token = mem_new0(usbtoken_t, 1);
-	ASSERT(token);
+	IF_NULL_RETVAL_ERROR(token, NULL);
```

Comment:
remove this (mem_new0 uses mem_alloc0 whisch asserts the return value is nonzero